app/Models/Champion.php

```diff
@@ -8,15 +8,30 @@ class Champion
     public $actualHp;
     public $maxHp;
     public $baseHp;
+    public $hpGrowth;
     public $armor;
+    public $armorGrowth;
     public $items = [];
+    public $magicResist;
+    public $magicResistGrowth;
+    public $level;
+    public $levelTreshold = [];

```

Comment:
jak to jest do ilosci doswiadczenia to ```experienceThreshold``` .

poza tym, to jest bodajze sztywne dla kazdej postaci, wiec lepiej miec to chyba w funkcji ktora te wartosci trzyma.